concert/app/src/functions/on_event_stage_changed/index.py

```diff
@@ -88,3 +104,107 @@ def get_song_list_update_transaction_item(message_data):
     }
 
     return {"Update": song_list_update_args}
+
+
+def get_song_list_get_transaction_item():
+    serializer = boto3.dynamodb.types.TypeSerializer()
```

Comment:
This could probably be a global object, to reduce memory usage and garbage collection.

Reply:
Done.